personio-personnel-data-api-oa3.yaml

```diff
@@ -351,7 +351,7 @@ paths:
         - Employees
       summary: Create an employee
       description: >
-        Creates new employee. Status of the employee will be set to `active` if `hire_date` provided is in the past. Otherwise status will be set to `onboarding`. This endpoint responds with `id` of created employee in case of success.
+        Creates new employee. If the status of the employee is not provided, it will be set based on `hire_date` value - if it is in the past, status will be `active`, otherwise `onboarding`. This endpoint responds with the `id` of created employee in case of success.
```

Comment:
```suggestion
        Creates a new employee. If the employee's status is not provided, it will be set based on the `hire_date` value - if it is in the past, status will be `active`, otherwise `onboarding`. This endpoint responds with the `id` of the created employee in case of success.
```